src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift

```diff
@@ -69,3 +69,9 @@ extension Locale {
 	}
 
 }
+
+extension Array where Element == Country {
+	var sortedByLocalizedName: [Country] {
+		return self.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
```

Comment:
```suggestion
		self.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
```

Just a hint: you can remove `return` if you want